app/main.py

```diff
@@ -1,3 +1,18 @@
-def cache(func):
-    # Write your code here
-    pass
+from typing import Any
+
+
+def cache(func: callable) -> callable:
+    _cache = {}
+
+    def some_inputs(*inputs: Any) -> str:
+        if func.__name__ not in _cache:
+            _cache[func.__name__] = {}
```

Comment:
you store **name of the function** -> you should **args**

Reply:
u use dict in dict - it's unnecessary

Reply:
> u use dict in dict - it's unnecessary

okay, I'll fix it. thx)